app/helpers/batch_edits_helper.rb

```diff
@@ -6,6 +6,6 @@ def batch_delete
   end
 
   def render_check_all
-    render partial: 'batch_edits/check_all' unless params[:controller].match("my/collections")
+    render partial: 'batch_edits/check_all' unless params[:controller] =~ /my\/collections/
```

Comment:
@awead In Sufia 7 we had to use an "==" to fix this because there seems to be a bug in Rubocop regarding this. See https://github.com/projecthydra/sufia/commit/c59b40f3648d2620b8a0a667d493fd0998b8413d